lib/sensors/MoistureSensor.cpp

```diff
@@ -0,0 +1,104 @@
+#include "mbed.h"
+#include "MoistureSensor.h"
+
+constexpr int Sensor_I2C_Address = 0x36 << 1;                               //MBED I2C uses 8 bit addressing, so addresses are left shifted by 1 (may need to be shifted by 2) 
```

Comment:
Why not just use const?

Reply:
constexpr is safer and should always be used when possible.